lib/explorer/series.ex

```diff
@@ -1241,6 +1241,27 @@ defmodule Explorer.Series do
 
   def mean(%Series{dtype: dtype}), do: dtype_error("mean/1", dtype, [:integer, :float])
 
+  @doc """
+  Gets the mode value of the series.
+
+  ## Supported dtypes
+
+    * `:integer`
+    * `:float`
+
+  ## Examples
+
+      iex> s = Explorer.Series.from_list([1, 2, 2, 3])
+      iex> Explorer.Series.mode(s)
+      2
+  """
+  @doc type: :aggregation
+  @spec mode(series :: Series.t()) :: float() | nil
```

Comment:
Float or int, right?

```suggestion
  @spec mode(series :: Series.t()) :: number() | nil
```